### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by cmsigrist**
Add a Profile page with a Settings page

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/blindly/SplashScreen.kt [hunk 1]
```diff
@@ -4,22 +4,28 @@ import android.content.Intent
...
         supportActionBar?.hide()
-        window.decorView.apply {
-            systemUiVisibility = View.SYSTEM_UI_FLAG_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_FULLSCREEN
-            }
 
+        val heart = findViewById<ImageView>(R.id.splashscreen_heart)
+        val beating = AnimationUtils.loadAnimation(this, R.anim.beating_heart)
+
+        // Use bounce interpolator with amplitude 0.2 and frequency 20
+        val interpolator = BounceInterpolator(0.3, 20.0)
```
Comment (r-ismer) [inline app/src/main/java/ch/epfl/sdp/blindly/SplashScreen.kt:25]: not the same constants

File: app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import android.view.View
+import android.widget.SeekBar
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import ch.epfl.sdp.blindly.R
+
+
+const val EXTRA_LOCATION = "user_location"
+const val EXTRA_SHOW_ME = "user_show_me"
+//const val REQUEST_LOCATION = 1
```
Comment (cgntnr) [inline app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt:14]: Is this comment still needed, also are these used outside, maybe they could be private ?

**Review by cgntnr — APPROVED** [review]
Added a note on a comment and making vals private when possible otherwise seems okay to me.

**Review by r-ismer — APPROVED** [review]
Seems very clean and clear to me, I approve!!

File: app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt (hunk 2 above)
Comment (cmsigrist) [reply app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt:14]: They're used by the children Activities, and the commented one will be used next week

Comment (cgntnr) [reply app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt:14]: All good, thanks!

**Pull request merged**